frontend/src/pages/util/login.js

```diff
@@ -1,20 +1,82 @@
 import { DATABASE } from "./data";
+import { checkCredentials } from "./userVerification";
+
+/***
+ * Handles user account creation.
+ * Returns true or false depending on status of user creation.
+ */
+export async function handleUserCreation(loginInfo) {
+  const {
+    firstName,
+    lastName,
+    userHandle,
+    email,
+    githubHandle,
+    password,
+    confirmPassword,
+  } = loginInfo;
+
+  if (
+    !Object.values(loginInfo).some(
+      (userInput) => userInput.replace(" ", "") === ""
+    )
+  ) {
+    if (!(email && email.includes("."))) {
+      // Invalid email address.
+      return;
+    }
+    if (password !== confirmPassword) {
+      // Passwords don't match.
+      return;
```

Comment:
Do we want to return nothing or return a boolean?

Reply:
Decided to return a boolean (false) for now, but when I get to focusing on CSS, I want a popup to appear.